Rakefile

```diff
@@ -0,0 +1,12 @@
+require 'rake'
+require 'rake/testtask'
+
+desc 'Test the Foreman Proxy plugin.'
```

Comment:
s/Foreman Proxy plugin./Prprocessor/g

Reply:
I may or may not have copy pasted something